src/widget/OktaSignIn.js

```diff
@@ -1,10 +1,15 @@
-/*global getRequireConfig */
+/*globals module */
 /*jshint unused:false */
 
 var OktaSignIn = (function () {
 
   var _ = require('underscore');
 
+  // Remove once these are explicitly required in Courage
```

Comment:
Should add a comment about why we need to do this. I.e. along the lines of "the wrappers are modifying jquery/underscore/handlebars modules".